### DISCUSSION TITLE
Add Text Annotation block to the Workflow Designer

### DISCUSSION THREAD

**Discussion by cmrusna97** [cortezaproject/corteza-server#1741] [Ideas]
Recommend adding the equivalent of the BPMN Text Annotation artifact as an available block in the Workflow designer, to allow designers to add remarks to their workflows.

This wouldn't be too difficult - just a string field and the ability to attach to an event block with a straight dotted line if desired.  No validation logic required.  The gray block in the image below is an example.

![image](https://github.com/cortezaproject/corteza/assets/26382972/84cc61aa-9fb3-460a-9671-60b89a0880b5)

**Comment by Fajfa** [comment]
Created an issue, closing discussion
https://github.com/cortezaproject/corteza/issues/1749
